API.md

```diff
@@ -125,10 +337,11 @@ frisby.create('Ensure response returns one cookie called "auth"')
 ```
 
 ### expectJSON([path], json)
+
 Tests that response body is JSON and [deeply equals](http://chaijs.com/api/bdd/#deep) the provided JSON.
 
-* Types: `path`: `string`, `json`: `JSON`
-* Defaults: `none`
+- Types: `path`: `string`, `json`: `JSON`
+- Defaults: `none`
```

Comment:
It's worth mentioning what `path` is, or else moving the "Using Paths" section before the first one of these.

Reply:
Added an extra bit to each method pointing the user at the Using Paths section.